src/redis-check-aof.c

```diff
@@ -333,6 +333,7 @@ int checkSingleAof(char *aof_filename, char *aof_filepath, int last_file, int fi
  * 1. The file is an old style RDB-preamble AOF
  * 2. The file is a BASE AOF in Multi Part AOF
  * */
+/* 判断是否是混合持久化 AOF 类型或者是增量 AOF 的 base 文件，根据前 5 个字符是否为 REDIS */
```

Comment:
我们就这样叫 7.0 里的 Multi Part AOF 吧（也是原作者和社区里接受了的叫法）
```suggestion
/* 判断是否是混合持久化 AOF 类型或者是 MP-AOF 的 base 文件，根据前 5 个字符是否为 REDIS */
```